modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/TxDeadlockDetection.java

```diff
@@ -285,7 +284,7 @@ private void map(@Nullable Set<IgniteTxKey> keys, Map<IgniteTxKey, TxLockList> t
 
             assert set;
 
-            if (nodeId == null || itersCnt++ >= DEADLOCK_MAX_ITERS || timedOut)
+            if (nodeId == null || itersCnt++ >= cctx.tm().deadlockMaxIters || timedOut)
```

Comment:
Let's invoke method instead accessing variable

Reply:
Let's keep it as is. 
This may cause refactoring in futire :)